CMakeLists.txt

```diff
@@ -95,12 +89,26 @@ else()
   set(UFS_CMAKE_BUILD_TYPE_FLAG "-DREPRO=ON")
 endif()
 
+
+if(UFS_APP MATCHES "^(ATM)$" OR UFS_APP MATCHES "^(ATMAERO)$" OR UFS_APP MATCHES "^(S2S)$")
+  set(UFS_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/ufs-weather-model)
+  set(UFS_REPO https://github.com/ufs-community/ufs-weather-model)
+  set(UFS_TAG "develop" )
+  set(patch_command ${CMAKE_COMMAND} -E copy_if_different ${CMAKE_CURRENT_SOURCE_DIR}/patches/mom6.patch <SOURCE_DIR>/MOM6-interface && cd <SOURCE_DIR>/MOM6-interface && git apply mom6.patch)
+else(UFS_APP MATCHES "^(NG-GODAS)$" )
```

Comment:
Is this still needed, given that we updated the soca branch?

Reply:
If we switch to the branch I used for the PR into the WM, we can probably combine all APPs together, though there will still be a patch required to get ng-godas to build. Since the patches tend to be kind of fragile though, it might be best to only apply the patch for UFS_APP=NG-GODAS.